cinema/models.py

```diff
@@ -1,9 +1,26 @@
 from django.db import models
 
 
+class Actor(models.Model):
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+
+
+class CinemaHall(models.Model):
+    name = models.CharField(max_length=255)
+    rows = models.IntegerField()
+    seats_in_row = models.IntegerField()
+
+
 class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
+    actors = models.ManyToManyField(Actor)
```

Comment:
what if we want to get the movies from the actor (reverse relationship)?

Reply:
we can use related_name